### ISSUE TITLE
Running out of space

### ISSUE THREAD

**Issue by szepeviktor** [s3ql/s3ql#245]
Some providers give you a fixed size S3 compatible bucket, e.g. @UpCloudLtd

> After compression:    843 MiB (26.60% of total, 28.73% of de-duplicated)
> Database size:        6.01 MiB (uncompressed)

How will I know bucket usage? Should I add these two numbers together?
It is hard to parse for a machine but could be done.

Thank you!

**Comment by d--j** [comment]
S3QL assumes that the object storage does not have a size limit. It might be dangerous to use S3QL with UpCloud's maximum of 1TB. 

S3QL does not tell you the exact bucket usage. These two number are only an estimate. The database gets saved compressed in object storage (with a very decent compression ration). S3QL keeps 10 backups of the compressed database around.

You probably should look at the authoritative place to get the current usage: [the UpCloud API](https://developers.upcloud.com/1.3/15-object-storage/#get-object-storage-details).

**Comment by szepeviktor** [comment]
Thank you!

> S3QL assumes that the object storage does not have a size limit.

@Nikratio Could you confirm that?

**Comment by Nikratio** [comment]
> Thank you!
> 
> > S3QL assumes that the object storage does not have a size limit.
> 
> @Nikratio Could you confirm that?

Yes.
